src/libstd/keyword_docs.rs

```diff
@@ -967,9 +967,59 @@ mod move_keyword {}
 //
 /// A mutable binding, reference, or pointer.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// `mut` can be used in several situations. The first is mutable bindings,
```

Comment:
while we did use "binding" for a while, these are "mutable variables". See https://doc.rust-lang.org/stable/book/ch03-01-variables-and-mutability.html (and a bunch of other times on this page I won't mention)